### PR TITLE
Migrate showcase to static pages

### PR DESCRIPTION
**Pull request by lmuntaner**
Migrate the showcase from dynamic `[page].astro` to static pages.

In this PR, we migrate the last of the pages and remove `[page].astro`.

What is missing now is to remove the iiPages list in `showcase.ts` to list the pages in the showcase's home page.

<hr/><details><summary>🟡 Some screens were changed</summary></details>

### CODE REVIEW

File: src/showcase/src/pages/loader.astro [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+import Screen from "../layouts/Screen.astro";
+---
+
+<Screen title="Loader" pageName="loader">
+  <script>
+    import { withLoader } from "$src/components/loader";
+    import { showMessage } from "$src/components/message";
+
+    // TODO: Fix loader image not loading
```
Comment (nmattia) [inline src/showcase/src/pages/loader.astro:10]: leftover TODO?

**Review by nmattia — COMMENTED** [review]
Thanks! Do you know why the screenshot changed?

File: src/showcase/src/pages/loader.astro (hunk 1 above)
Comment (lmuntaner) [reply src/showcase/src/pages/loader.astro:10]: This is currently in `main`. I'll fix it in another PR.

**Comment by lmuntaner** [comment]
> Thanks! Do you know why the screenshot changed?

I added two more toasts: one for success one for info. There were only error toasts before.

I should have added it in the description. Sorry.

**Review by nmattia — APPROVED** [review]

**Pull request merged**